doc/examples/segmentation/plot_chan_vese.py

```diff
@@ -0,0 +1,70 @@
+"""
+======================
+Chan-Vese Segmentation
+======================
+
+The Chan-Vese Algorithm is designed to segment objects without clearly
+defined boundaries. This algorithm is based on level sets that are
+evolved iteratively to minimize an energy, which is defined by
+weighted values corresponding to the sum of differences inintensity
+from the average value outside the segmented region, the sum of
+differences from the average value inside the segmented region, and a
+term which is dependent on the length of the boundary of the segmented
+region.
+
+This algorithm was first proposed by Tony Chan and Luminita Vese, in
+a publicaion entitled "An Active Countour Model Without Edges" [1]_.
+
+This implementation of the algorithm is somewhat simplified in the
+sense that the area factor 'nu' described in the original paper is not
+implemented, and is only suitable for grayscale images.
+
+Typical values for lambda1 and lambda2 are 1. If the 'background' is
+very different from the segmented object in terms of distribution (for
+example, a uniform black image with figures of varying intensity), then
+these values should be different from each other.
+
+Typical values for mu are between 0 and 1, though higher values can be
+used when dealing with shapes with very ill-defined contours.
+
+The algorithm also returns a list of values which corresponds to the
+energy at each iteration. This can be used to adjust the various
+parameters described above.
```

Comment:
^ How about putting these great description and recommendations to the `Notes` section of the function docstring?